index.html

```diff
@@ -94,14 +94,71 @@ <h1 class="primary-slider__title font-secondary">Beautiful Escape</h1>
 
             <div class="gallery__content">
                 <p class="gallery__text font-tertiary"> By better understanding the various aspects of surfing, you will improve faster and have more fun in the water. </p>
-                <a class="read-more font-primary" href="#">Read More</a>
+                <a class="primary-button font-primary" href="#">Read More</a>
 
             </div>
 
         </article>
 
     </div>
 
+    <div class="shop-container">
+        <div class="shop-heading">
+            <span class="heading__subtitle font-primary">shop</span>
+            <h2 class="heading__title font-secondary">Surfboards</h2>
+        </div>
+
+        <div class="shop-grid group">
+
+            <div class="shop-grid__col-01 left">
+                <img class="catalog__surf-image" src="images/surf-1.png"  alt="slider board" />
+                <div class="catalog__content">
+                     <span class="content__subtitle font-tertiary">funboards</span>
+                     <h3 class="content__title font-secondary">Chilli Rare Bird</h3>
+
+                     <span class="content__price font-tertiary">$890</span>
+                     <a class="content__buy font-primary">buy</a>
+                </div>
+            </div>
+
+            <div class="shop-grid__col-02 left">
+                <img class="catalog__surf-image" src="images/surf-2.png"  alt="slider second board" />
+                <div class="catalog__content">
+                     <span class="content__subtitle font-tertiary">Surfboards</span>
+                     <h3 class="content__title font-secondary">Emery NEM XF</h3>
+
+                     <span class="content__price font-tertiary"">$950</span>
+                     <a class="content__buy font-primary">buy</a>
+                </div>
+            </div>
+
+            <div class="shop-grid__col-03 left">
+                <img class="catalog__surf-image" src="images/surf-3.png"  alt="slider third board" />
+                <div class="catalog__content">
+                     <span class="content__subtitle font-tertiary">Funboards</span>
+                     <h3 class="content__title font-secondary">Agency GROM</h3>
+
+                     <span class="content__price font-tertiary"">$670</span>
+                     <a class="content__buy font-primary">buy</a>
+                </div>
+            </div>
+
+        </div>
+
+            <a class="primary-button font-primary" href="#">show all</a>
+
+            <div class="shop-controls">
+                <svg class="shop-controls__previous-arrow" xmlns="http://www.w3.org/2000/svg" width="11" height="19" viewBox="0 0 11 19">
+                 <path fill="#333" d="M.15 9.64l8.96-8.49c.2-.2.55-.2.76 0 .2.2.2.52 0 .72L1.3 10l8.57 8.13c.2.2.2.52 0 .72-.1.1-.25.15-.38.15a.55.55 0 0 1-.38-.15L.16 10.36a.5.5 0 0 1 0-.72z"/>
+                </svg>
```

Comment:
Probably the parent `.shop-controls` div is unnecessary here. Plus, try to use a <button> tag (with the SVG of the arrows inside).